docs/config/pipeline_config.md

```diff
@@ -109,3 +109,60 @@ data_vars:
       units: deg
       comment: This is a brand new variable called 'wind_dir'
 ```
+
+## Adding a New Pipeline
+
+### Creating a New `pipeline.yaml` File
+
+When working with an existing ingest, you might need to create a new `pipeline.yaml` file to accommodate different configurations. For example, you may want to set up pipelines for different sites, apply different metadata, or handle other specific processing needs. Adding a new `pipeline.yaml` file allows you to maintain organization and flexibility within your project.
+
+#### Reasons for Adding a New `pipeline.yaml` File
+
+- **Different Site Configurations:** If you're processing data from multiple sites, each site may have unique settings or parameters. Creating separate `pipeline.yaml` files for each site ensures that these configurations are handled properly.
+- **Varying Metadata:** Different datasets might require distinct metadata configurations. Separate `pipeline.yaml` files can help manage these variations efficiently.
+- **Specialized Processing:** You might need different processing steps or quality control measures for different datasets. Multiple `pipeline.yaml` files allow you to customize these processes without disrupting the overall pipeline structure.
+
+#### Suggested Naming Conventions
+
+To keep your project organized, consider adopting a clear and consistent naming convention for your `pipeline.yaml` files. Here are some suggestions:
+
+- **Site-Specific Pipelines:**
+  - `pipeline_sgp.yaml` for the Southern Great Plains site.
+  - `pipeline_nsa.yaml` for the North Slope of Alaska site.
+
+- **Metadata Variations:**
+  - `pipeline_metadata_v1.yaml` for the first version of metadata.
+  - `pipeline_metadata_alt.yaml` for an alternative metadata configuration.
+
+- **Specialized Processing:**
+  - `pipeline_qc.yaml` for a pipeline focused on quality control.
+  - `pipeline_transform.yaml` for pipelines that require specific data transformations.
```

Comment:
This isn't rendering right on readthedocs:

<img width="694" alt="image" src="https://github.com/user-attachments/assets/8be57eeb-ca50-480e-ae2b-fd9803172b0d">

I think you need to insert an empty line whenever you go from a main bullet point to a sub bullet point line. 

E.g.,

```md
- **Site-Specific Pipelines:**

  - `pipeline_sgp.yaml` for the Southern Great Plains site.
  - `pipeline_nsa.yaml` for the North Slope of Alaska site.
```